src/sb_pointcloud_processing/include/ColourspaceConverter.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by: Valerian Ratu
+ * Created on: 2018/01/13
+ * Description: A class which converts RGB pointclouds to the HSV colourspace.
+ *              Used in the rgb_to_hsv nodelet.
+ */
+
+#ifndef PROJECT_PREPROCESSFILTER_H
+#define PROJECT_PREPROCESSFILTER_H
```

Comment:
We should rename `PROJECT_PREPROCESSFILTER_H` to something a bit more descriptive, like `SB_POINTCLOUD_PROCESSING_COLORSPACE_CONVERTER_H` (bit long, but following our general convention elsewhere). We should make sure that all header files have descriptive/unique preprocessor guards, I'm just noting this one instance, there are a few more.

Reply:
:heavy_check_mark: